__tests__/app.test.js

```diff
@@ -71,3 +70,19 @@ describe('/api/articles/:article_id', () => {
   });
 });
 
+describe('/api/articles', () => {
+  test('GET:200 gets all articles with specified columns', () => {
+    return request(app)
+      .get('/api/articles/')
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.articles.length).toBe(13);
+        expect(body.articles).toBeSortedBy('created_at');
```

Comment:
Well done for remembering that the articles need to be default sorted! As we're looking for a different behaviour here we could separate this into its own test